proof-libs/fstar/rust_primitives/Rust_primitives.Hax.fst

```diff
@@ -82,3 +82,6 @@ let while_loop #acc_t
     else current in 
   while_loop_internal init
   
+
+/// Represents backend failures
+let failure : string -> string -> unit = fun _ _ -> ()
```

Comment:
We chatted about it live, let's do:
```fstar
let failure #t (_error: string) (_ast: string): Pure t False (fun _ -> True) = ()
```

Also, that function can actually be a value of any type. Ah and this represent is more a engine failure than a backend failure.